src/lib/config/index.ts

```diff
@@ -141,11 +145,13 @@ export const SESSION_SECRET =
 	'dcOVe-ZW3ul77l23GiQSNbTJtMRio87G2yUOUAk_otcbL3uywfyLMZ9NBmDMuuOt'
 
 export const XAPI = set({
-	activityBaseUri: 'http://cslearning.gov.uk/courses',
 	auth: {
 		password: env.XAPI_PASS || '1c0e1b6827606d7efed71e204939d048f94f842b',
 		username: env.XAPI_USER || '66f2b4fc001e3da992d23b57d8a7457655bea078',
 	},
+	courseBaseUri: 'http://cslearning.gov.uk/courses',
```

Comment:
Should these URIs be singular, i.e. `/course` instead of `/courses`?

Reply:
I prefer plural on as requesting a /courses on an API would return a list, not a single course. Convention in xAPI is inconsistent so went with my preference :).